src/stache/includes/components/tabs-sitelevel.html

```diff
@@ -0,0 +1,19 @@
+<p>
+  If you use tabs or vertical tabs as page navigation, each one should have an entry in the browser history with a distinct title. This enables users to select the browser's back button to return to the previous view. Tabs and vertical tabs that are used as site-level navigation contain all of a page's content and can't be dismissed.
```

Comment:
I'd change "each one" to "each tab." ... But I wonder if there is a way to reword the portion of the first sentence after the comma to make it a command and avoid "should," which could make it sound optional. Maybe "provide an entry for each tab in the browser history with a distinct title"? What do you think?